app/models/card.rb

```diff
@@ -20,4 +20,7 @@ def original_translated_text_equal
     def full_downcase(str)
       str.mb_chars.downcase.to_s.squeeze(' ').lstrip
     end
+
+    scope :review_date_card, -> { where('review_date <= ?', Time.now.strftime('%Y-%m-%d')).
+        select(:id, :original_text).order('RANDOM()').first }
```

Comment:
Expression at 25, 61 should be on its own line.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.